en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md

```diff
@@ -113,16 +113,149 @@ Provide the log level as trailing arguments for the envoy command as follows.
+    }
+  }
+  headers {
+    header {
+      key: "x-wso2-cluster-header"
+      value: "carbon.super_clusterProd_localhost_SwaggerPetstore1.0.6"
+    }
+  }
+  headers {
+    header {
+      key: "X-TRACE-KEY"
+      value: "3973774370179828478"
+    }
+  }
+  headers_to_remove: "api_key"
+  headers_to_remove: "internal-key"
+}
+dynamic_metadata {
+  fields {
+    key: "keyType"
+    value {
+      string_value: "PRODUCTION"
+    }
+  }
+  fields {
+    key: "token"
+    value {
+      string_value: "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ik1UWm1ZemsxWlRJeVkyTmlNR00wT1RJMFpqTXdNRE5rTldZek16QTJOVEl4TWpRelpERTFNdz09In0.eyJhdWQiOiJodHRwOlwvXC9vcmcud3NvMi5hcGltZ3RcL2dhdGV3YXkiLCJzdWIiOiJhZG1pbiIsInNjb3BlIjoicmVhZDpwZXRzIiwiaXNzIjoiaHR0cHM6XC9cL2xvY2FsaG9zdDo5MDk1XC90ZXN0a2V5Iiwia2V5dHlwZSI6IlBST0RVQ1RJT04iLCJleHAiOjE2NDk2Njg2NjUsImlhdCI6MTY0OTY2NTA2NSwianRpIjoiZDI3MTRiYmMtMWE5OC00ODc5LThmM2QtM2I5ZGYwNjI1NDcwIn0.MXkSAq98Zx32j5ijK4MLvlocMhjVW_xxW3ECeGVmHaRkqg43bO2PLtdoh952YBbK4xzK3mDhAxF1VxynDyD5u1qpu696e4w3UwF2pfU3hAdbG2fDdcm82KiSfydawOnivxFr-QX08tQxDTO9i5Fgo7nZMsZkNRN2er1asctDaDRcqcR00Jh6h1jyqBU6Ixf67dkZk61et8yNdGeSiuy-QXLVHsN8YbvXNyct1d1nVjZrE3d_kDkk_ZQlqHj327BXolCl_tiinj_bBDovmqIDNwTXkL47rtuWX6sB1uBJ0xBHtuZxT7-IgB1WUVvVcbAqNebQFs1bV8A-h0efQKKBdg"
+    }
+  }
+  fields {
+    key: "tokenType"
+    value {
+      string_value: "JWT"
+    }
+  }
+  ...
+  ...
+```
+
+Router to client response headers
+```yaml
+[2022-04-11 08:18:04.956][15][debug][http] [source/common/http/conn_manager_impl.cc:1467] [C19][S3973774370179828478] encoding headers via codec (end_stream=false):
+':status', '200'
+'date', 'Mon, 11 Apr 2022 08:18:04 GMT'
+'content-type', 'application/json'
+'access-control-allow-origin', '*'
+'access-control-allow-methods', 'GET, POST, DELETE, PUT'
+'access-control-allow-headers', 'Content-Type, api_key, Authorization'
+'server', 'envoy'
+'x-envoy-decorator-operation', 'localhost:^/v2/pet/findByStatus(\?([^/]+))?$'
+```
+
+Follow [Command line options]({{envoy_path}}/operations/cli) for more information.
```

Comment:
Not sure what this link is. @pubudu538 is this how this is supposed to be?

Reply:
Choreo Connect is built on top of Envoy. That is the reason for having this link.